hack/util.sh

```diff
@@ -368,6 +406,64 @@ function validate_response {
 	return 1
 }
 
+# start_etcd starts an etcd server
+# $1 - Optional host (Default: 127.0.0.1)
+# $2 - Optional port (Default: 4001)
+function start_etcd_extended() {
```

Comment:
Ok, I recognize this function.  It shouldn't be `_extended`, if this can't support the general case of starting etcd, then it belongs in your extended test.  Since this very similar to `test-integration.sh`, I'm ok with having such a function here, but I'd say simplify it.

 1. Use "standard" env vars in this file to start it (similar to starting the openshift server).
 1. Don't inspect currently running processes.  That was probably a mistake in the first script.
 1. You don't need to check etcd version, since we only support v2 now.